src/Fable.React.Props.fs

```diff
@@ -893,9 +893,9 @@ type UserSelectOptions =
     | Unset
 
 type CSSProp =
-    | AlignContent of AlignContentOptions
-    | AlignItems of AlignItemsOptions
-    | AlignSelf of AlignSelfOptions
+    | [<CompiledName "align-content">] AlignContent of AlignContentOptions
+    | [<CompiledName "align-items">] AlignItems of AlignItemsOptions
```

Comment:
React seems to be disagreeing with this one. https://github.com/fable-compiler/fable-react/issues/182

Reply:
Yes, I should not have touched the CssProps. The CompiledName is only required for Options.